lib/internal/cluster/child.js

```diff
@@ -228,16 +228,23 @@ function _disconnect(masterInitiated) {
 
 // Extend generic Worker with methods specific to worker processes.
 Worker.prototype.disconnect = function() {
-  _disconnect.call(this);
+  if (![ 'disconnecting', 'destroying' ].includes(this.state)) {
```

Comment:
lgtm
nit: remove the spaces could be a better style.

Reply:
Which spaces? The ones in the array are enforced by the linter.

Reply:
I thought it would be `[‘disconnecting’, ‘destroying’]`. That’s ok. Forgot about this